src/main/java/seedu/duke/data/notebook/Notebook.java

```diff
@@ -46,9 +49,24 @@ public void addNote(Note note) {
 
     /**
      * Removes a note from the timetable.
+     *
+     * @throws SystemException if an error occurs.
      */
-    public void deleteNote(int note) {
-        notes.remove(note);
+    public void deleteNote(int index) {
+        notes.remove(index);
+    }
+
+    /**
+     * Removes a note from the timetable.
+     *
+     * @throws SystemException if an error occurs.
+     */
+    public void deleteNote(String title) {
+        notes.removeIf(note -> note.getTitle().equals(title));
+    }
```

Comment:
I don't see the System Exception being thrown in the method?

Reply:
Dont need. I have catch possible errors before it adds.

Reply:
Removed throws comment